### PR TITLE
Javascript Group Capstone Project

### PR DESCRIPTION
**Pull request by Bupilipili**
**In this capstone, we implemented the following as per the project requirements:**

**The home page**.

> When the page loads, the web app retrieves data from:
> [Meals DB](https://www.themealdb.com/api.php) and display meals on the homepage.
> The [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to deliver the item likes.

> When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.

**The comments popup.**

> When the user clicks on the Comments button, the comments popup appears and the web app retrieves data from:
> The base API shows details about the selected item.
> The Involvement API to deliver the item comments.
> When the user clicks the Comment button, the data is recorded in the Involvement API, and the screen is updated.
> To make the website dynamic and build basic single-page app:

**General requirements checklist.**

- Used ES6 syntax and modules.
- Used callbacks and promises.
- Used webpack.
- Used AAA pattern for unit tests with Jest.
- The home page header/footer and navigation are similar to the given mockup.
- Resolved simple git conflicts.
- Perform code reviews for team members' pull requests.
- Updated Readme file with video presentation link.
-  No linters error.
-  Used the correct Gitflow workflow.
-  documented work professionally.
-  Followed the best practices of HTML, CSS, and JavaScript.

### CODE REVIEW

File: src/modules/popupCard.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/* eslint-disable linebreak-style */
```
Comment (elisha2kyakpo1) [inline src/modules/popupCard.js]: - [x] Kindly do not disable linters, linters are used to highlight errors in our code. The errors should be fixed to have a clean code that meets standards.

File: src/testing/itemsCounter.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/no-extraneous-dependencies */
+// Import the function to test
```
Comment (elisha2kyakpo1) [inline src/testing/itemsCounter.test.js]: Kindly do not disable linters, linters is used to highlight errors in our code. The errors should be fixed to have a clean code that meets standards.

**Comment by Bupilipili** [comment]
Note: To run this project on your local device you are required to erase all the public path in webpackconfig.js file into this(public path"./") since if the public path remains like this the deployment doesn't show the javascript and the CSS of the code.

File: README.md [hunk 3]
```diff
@@ -1 +1,148 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🌍 Online Version <a name="online-version"></a>
+  
+  [Live version](https://anshuman7negi.github.io/Kanban_board/dist/)
+
+## 📹 Video Presentation <a name="Video-Presentation"></a>
+  
+  [Video Presentation](https://www.loom.com/share/c07159866c5f4b279682939afeb165be)
+
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] Great job recording the video, unfortunately, it did not meet the requirements for this project. Kindly record another video and remember to follow all the requirements for the video.


video

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Bupilipili, @anshuman7negi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ Tests passing👏
👏 The app works as expected.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/popupCard.js (hunk 1 above)
Comment (Bupilipili) [reply src/modules/popupCard.js]: Alright i have fixed it.

File: src/testing/itemsCounter.test.js (hunk 2 above)
Comment (Bupilipili) [reply src/testing/itemsCounter.test.js]: I also fixed it.

**Review by LionRouge1 — APPROVED** [review]
Hi **Team**, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**